api/object.go

```diff
@@ -324,6 +325,9 @@ func (opts GetObjectOptions) Apply(values url.Values) {
 	if opts.Marker != "" {
 		values.Set("marker", opts.Marker)
 	}
+	if opts.OnlyMetadata {
+		values.Set("onlymetadata", "true")
```

Comment:
Since we're introducing a new query string param, are we doing all lower case or camelcase? We have to change a bunch of them anyway when going to v2.0 but if we decide now and stick with it it's one change we can avoid.

Reply:
I vote all lowercase btw

Reply:
Yeah I was also leaning towards all lowercase